xoai-data-provider/src/main/java/io/gdcc/xoai/dataprovider/handlers/GetRecordHandler.java

```diff
@@ -98,6 +97,7 @@ public GetRecord handle(OAICompiledRequest parameters) throws OAIException, Hand
                 for (About about : item.getAbout())
                     record.withAbout(about);
             }
+            */
```

Comment:
Do you remember *why* you attached the "raw" metadata instead of the transformed version?